src/state/actions/window.js

```diff
@@ -117,6 +116,18 @@ export function closeCompanionWindow(windowId, companionWindowId) {
   };
 }
 
+/**
+* Move companion window between right and bottom area
+* @param {String} companionWindowId
+*/
+export function toggleAreaOfCompanionWindow(companionWindowId) {
+  return (dispatch, getState) => {
+    const position = getState()
+      .companionWindows[companionWindowId].position === 'right' ? 'bottom' : 'right';
+    dispatch(updateCompanionWindow(companionWindowId, { position }));
```

Comment:
It looks like we may be missing a unit test for this function, would it be possible/easy to add one?